src/warpy_sync/syncer_deposit.go

```diff
@@ -145,7 +145,7 @@ func (self *SyncerDeposit) checkTx(tx *types.Transaction, block *BlockInfoPayloa
 					return nil
 				}
 
-				if slices.Contains(self.Config.WarpySyncer.SyncerDepositFunctions, method.Name) {
+				if slices.Contains(self.Config.WarpySyncer.SyncerDepositFunctions, method.RawName) {
```

Comment:
Name has been working for all the previous integrations, is RawName a standard field which always (or usually :)) exists?

Reply:
Name returns something like 'provide0' whatever that is...
And RawName is just the actual function name like 'provide'